src/repository/default.rs

```diff
@@ -239,10 +239,16 @@ impl Repository for $repo {
 
         // Create columns
         // TODO this breaks if we have more than (bind limit) columns
-        if !schema.arrow_schema.fields().is_empty() {
+        if !schema.fields().is_empty() {
             let mut builder: QueryBuilder<_> =
                 QueryBuilder::new("INSERT INTO table_column(table_version_id, name, type) ");
-            builder.push_values(schema.to_column_names_types(), |mut b, col| {
+
+            let fields: Vec<(String, String)> = schema.fields()
+                .iter()
+                .map(|f| (f.name().clone(), field_to_json(f).to_string()))
+                .collect();
```

Comment:
Wait, it's already implemented in https://github.com/splitgraph/seafowl/blob/3d428c98bd8b83237a3cb6cae1bb48080a3c4026/src/schema.rs#L37-L43, why the duplication? could pull that out into a function that `schema::Schema` and this both call if you want to not use the `schema::Schema` wrapper struct.